### ISSUE TITLE
Suggestion change some names

### ISSUE THREAD

**Issue by rushercrack** [THEREALWWEFAN231/wwe#298]
**Minecraft Version**
All
**What is the suggestion**
Change Anti-Knockback to Velocity and put it in Movement
Change the color of the WWE text
Change the message when trying to enable CustomFont while not having selected a font to: "It appears you haven't set a custom font yet. To do so, type .customfont and select a font of choice.
Change .toggle message to ".toggle <module>"
Change invalid toggle message to "That module does not exist."
**Why it will be useful**

**Comment by THEREALWWEFAN231** [comment]
Change the color of the WWE text?

**Comment by rushercrack** [comment]
Yes, it's a dark red currently which is hardly readable for some people. Maybe a option to choose the color yourself?

**Comment by Paralusion** [comment]
You can now change the watermark color
